locale/core.yml

```diff
@@ -217,6 +219,41 @@ core:
       remove_button: => core.ref.remove
       upload_button: Choose an Image...
 
+    users:
+      title: => core.ref.users
+      description: A paginated list of all users on your forum.
+
+      total_users: "Total users: {count}"
+
+      grid:
+        invalid_column_content: Invalid
+
+        default_columns:
+          id: ID
+
+          username: => core.ref.username
+          profile_link_tooltip: Visit {username}'s profile
+
+          join_time: Joined
+
+          group_badges: Groups
+          group_badges_none: None
+
+          email: Email
+          email_visibility_show: Show email address
+          email_visibility_hide: Hide email address
+          # Fake email address which is blurred before being replaced by the real email
+          email_hidden: "abcdefg12345@gmail.com"
+
+          edit_user: => core.lib.edit_user.title
```

Comment:
If referencing another key, we should use `ref` instead, lib is for string that are called directly from either forum or admin frontends.

So let's create a `core.ref.edit_user.title` and change the rest accordingly.

Reply:
Can we also replace `gmail.com` with `example.com`? I think It would be best to use something more generic instead of being brand specific to avoid any future headaches :)